templates/ticket/ticket.html

```diff
@@ -357,7 +357,13 @@
                     </div>
                 </div>
                 <div class="info-box">
-                    <div class="info-title">{{ _('Assignees') }}</div>
+                    <div class="info-title">
+                        {% trans trimmed count=assignees|length, singular=_("Assignee"), pluralized=_("Assignees") %}
```

Comment:
See #1554 ([line](https://github.com/DMOJ/online-judge/pull/1554/files#diff-742f815f437aaa2ea7933ce5bbed8efbR67)) it looks cleaner.

Reply:
I tried something similar but the resulting change broke the translations :/